components/button/src/fixtures.js

```diff
@@ -48,8 +50,10 @@ const ButtonWacky = () => (
 const ButtonAsLink = () => (
   <MemoryRouter>
     <div>
-      <Button as={Link} to="/">Home Link button</Button><br />
-      <Button as={Link} to="/test">Test Link button</Button><br />
+      <Button as={Link} to="/">Home Link button</Button>
+      <br />
+      <Button as={Link} to="/test">Test Link button</Button>
+      <br />
```

Comment:
I know this change is because of the `eslint --fix` but is using `<br />` something that we actively recommend instead of an `mb` prop on `Button`

Reply:
`Button` uses `display: inline-block` on desktop so you need to use `<br />` to put things onto new lines

this is as per govuk-frontend